sidechain/consensus/aura/src/lib.rs

```diff
@@ -233,14 +235,14 @@ fn proposing_remaining_duration<PB: ParentchainBlock>(
 	std::cmp::min(slot_remaining, proposing_duration)
 }
 
-fn authorities<C, P, B>(
-	ocall_api: &C,
-	header: &B::Header,
+fn authorities<ValidateerFetcher, P, ParentchainHeader>(
+	ocall_api: &ValidateerFetcher,
+	header: &ParentchainHeader,
 ) -> Result<Vec<AuthorityId<P>>, ConsensusError>
 where
-	C: ValidateerFetch + GetStorageVerified,
+	ValidateerFetcher: ValidateerFetch + GetStorageVerified,
 	P: Pair,
-	B: ParentchainBlock<Hash = H256>,
+	ParentchainHeader: ParentchainHeaderTrait<Hash = H256>,
```

Comment:
As only the header is used in this function, I removed the Block input enforcement.

Reply:
I think that's fine, yes 👍